yt/funcs.py

```diff
@@ -429,10 +429,10 @@ def is_root():
     """
     from yt.config import ytcfg
 
-    cfg_option = "__topcomm_parallel_rank"
+    cfg_option = "topcomm_parallel_rank"
     if not ytcfg.get("yt", "internals", "parallel"):
         return True
-    if ytcfg.get("yt", cfg_option) > 0:
+    if ytcfg.get("yt", "internals", cfg_option) > 0:
         return False
     return True
```

Comment:
Couldn't be simplified as (assuming above-mentioned change is accepted)
```suggestion
    return ytcfg.get("yt", "internals", "topcomm_parallel_rank") == 0
```
unless `ytcfg.get("yt", "internals", "topcomm_parallel_rank")` can take negative values or non-integers?

Reply:
I don't think so. Good suggestion.